.eslintrc.json

```
@@ -1,17 +1,14 @@
 {
-  "parser": "@typescript-eslint/parser",
-  "extends": ["next/core-web-vitals", "prettier", "plugin:import/recommended", "plugin:import/typescript"],
-  "plugins": ["prettier", "@typescript-eslint"],
+  "extends": ["@seoltab/eslint-config-web"],
+  "parserOptions": {
+    "project": "./tsconfig.json"
+  },
+  "ignorePatterns": [
+    ".next/*",
+    ".husky/*"
+  ],
```

```suggestion
  "ignorePatterns": [
    ".next/*",
    ".husky/*"
  ],
```
이건 위 .eslintignore에서 정의 된 부분이라 지워도 될거 같습니다

---

파일 두 개 쓸 필요 없을 것 같아서 그 부분을 살리고 .eslintignore 를 삭제하였음다.